src/org/jmc/ObjExporter.java

```diff
@@ -401,23 +401,28 @@ public static void export(@CheckForNull ProgressCallback progress, boolean write
 		}
 	}
 	
+	/**
+	 * Add CommonMCOBJ Header.
+	 * <p>
+	 * CommonMCOBJ is a common standard for exporting metadata
+	 * in OBJs.
+	 * @param objWriter
+	 *      The writer that's writing the OBJ file
+	 */
 	private static void writeCommonMcObjHeader(PrintWriter objWriter) {
 		objWriter.println("# COMMON_MC_OBJ_START");
-		objWriter.println("# version: 0");
+		objWriter.println("# version: 1");
 		objWriter.println("# exporter: jmc2obj");  // Name of the exporter, all lowercase, with spaces substituted by underscores
-		objWriter.println("#");
 		objWriter.println("# world_name: " + Options.worldDir.getName());  // Name of the source world
 		objWriter.println("# world_path: " + Options.worldDir.toString());  // Path of the source world
-		objWriter.println("#");
-		objWriter.println("# exported_bounds_min: " +  String.format("%d, %d, %d", Options.minX, Options.minY, Options.minZ));  // The lowest block coordinate exported in the obj file
-		objWriter.println("# exported_bounds_max: " + String.format("%d, %d, %d", Options.maxX-1, Options.maxY-1, Options.maxZ-1));  // The highest block coordinate exported in the obj file
-		objWriter.println("#");
+		objWriter.println("# export_bounds_min: " +  String.format("(%d, %d, %d)", Options.minX, Options.minY, Options.minZ));  // The lowest block coordinate exported in the obj file
+		objWriter.println("# export_bounds_max: " + String.format("(%d, %d, %d)", Options.maxX-1, Options.maxY-1, Options.maxZ-1));  // The highest block coordinate exported in the obj file
+		objWriter.println("# export_offset: " + String.format("(%d, 0, %d)", Options.offsetX, Options.offsetZ));
```

Comment:
@mmdanggg2 just want to be sure, would `Options.offsetX` and `Options.offsetZ` be correct in this case?

Reply:
Not quite, it's fine I'll merge this and make the necessary changes.